src/components/filters.jsx

```diff
@@ -0,0 +1,257 @@
+import React, { useContext } from 'react'
+import { FilterContext } from '../frontend/context/filterContext';
+
+export default function FilterSection() {
+
+    const { products, dispatch, filteredProducts, filteredList } = useContext(FilterContext)
```

Comment:
you can export a `useFilter` function from your filter context so that you don't have to import the `useContext` everywhere

Reply:
oh yes, right. Thank you, I'll update it .